src/libtls/tests/suites/test_socket.c

```diff
@@ -559,8 +651,26 @@ Suite *socket_suite_create()
 #define add_tls_curve_test(func) \
 	tcase_add_loop_test(tc, func, 0, tls_crypto_get_supported_curves(NULL));
 
+#define add_tls_versions_test(func, from, to) \
+	tcase_add_loop_test(tc, func, from, to);
```

Comment:
Like this the macro is not necessary, there is no benefit over calling `tcase_add_loop_test()` directly. However, you could perhaps change it to 
```suggestion
	tcase_add_loop_test(tc, func, from, to+1);
```
Then the versions you use below would actually be those tested.